src/Auth0Client.ts

```diff
@@ -931,6 +942,15 @@ export default class Auth0Client {
       if (e.message === MISSING_REFRESH_TOKEN_ERROR_MESSAGE) {
         return await this._getTokenFromIFrame(options);
       }
+      // A refresh token was found, but is it no longer valid.
+      // Fallback to an iframe.
+      if (
+        e.message &&
+        e.message.indexOf(INVALID_REFRESH_TOKEN_ERROR_MESSAGE) > -1
+      ) {
+        return await this._getTokenFromIFrame(options);
```

Comment:
This introduces an additional request in Safari where we know up front it will return `login_required`, while for other browsers it might still work.

I do not think the SDK is currently tracking which browsers can use iframe, nor do I think it should so I wonder what you think about this extra request.

Reply:
I think it's fine to leave it, we don't do browser detection for this elsewhere and it will work for other browsers that also block third-party cookies.